packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/BankDetails/template.tsx

```diff
@@ -91,7 +91,8 @@ const Template: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             {bankAccountName}
           </Text>
           <Text size='24px' color='grey600' weight={500}>
-            {achDetails && achDetails.bankAccountType.toLowerCase()}{' '}
+            {(achDetails && account?.details?.bankAccountType?.toLowerCase()) ||
```

Comment:
I wonder if this won't work for ACH accounts?

Reply:
we can improve, idk tbh